OpenOversight/app/utils.py

```diff
@@ -361,17 +362,24 @@ def filter_by_form(form_data, officer_query, department_id=None):
         ]
 
         if "Not Sure" in form_data["unit"]:
-            # Convert "Not Sure" to None, so the resulting SQL query allows NULL values
-            form_data["unit"].append(None)
+            include_null_unit = True
```

Comment:
Ha, looks like I introduced this in #91 😅

Reply:
It totally makes sense to think that sqlalchemy would automagically handle querying a list containing None/null but sql is hell so :shrug: